### ISSUE TITLE
Suggestion: Renaming Class "Character"

### ISSUE THREAD

**Issue by MeFisto94** [jvpichowski/ZayES-Bullet#3]
Hey,
This suggestion isn't important right now, but something to consider before releasing Version 1.0:

Since `java.lang` is always imported in each class, you can't directly reference the Character Component, because it matches the boxed Version of char. That's why you have:
```java
edS.getEntityData().setComponents(playerEntity,
                new com.jvpichowski.jme3.es.bullet.character.Character(50, 0.35f, 2f),
```
If you would change that to `PhysicsCharacter` or something, this would have been solved (Actually you might come up with different Components anyway.).

Another possibility would be to append `Component` to the class name of any Component, but I guess this is bad style :)

**Comment by jvpichowski** [comment]
Thanks for the issue report.

As you can see here it is possible to use the `Character` class with a direct reference:

https://github.com/jvpichowski/ZayES-Bullet/blob/master/src/test/java/com/jvpichowski/examples/es/bullet/character/CharacterTest.java#L75
Maybe you have messed something up with your imports.

Nevertheless I think `PhysicsCharacter` is a better name because the user has probably his own `Character`component class. 

I personally don't like this style because it is clear that you only attach components to an entity and not other classes. But I'm open for code style discussions. Let's discuss it in the forum because there are more experienced people than me.

**Comment by jvpichowski** [comment]
Fixed with commit 5832ca2e0538ebd211b1887387420cf2e00ea397

**Comment by MeFisto94** [comment]
Well it depends on your personal preference, I guess. I've used "Component" because that way I always know that it's a Component and no Control etc.

Two comments on that commit though:
A) It would be better to not merge changes into the same commit, in case you want to revert some changes or maybe want to find out where a particular bug came from.
B) If you name the commit `Fixes #3 - Renaming Character` (The #3 and maybe the Fixes are the important parts), then github will automatically close the issue and post the related commit (And the #3 is clickable in the commit message then) :)

And thanks for your work so far :)
